pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -290,6 +300,61 @@ func (c *Cluster) startMons(targetCount int) error {
 	return nil
 }
 
+func (c *Cluster) configureStretchCluster(mons []*monConfig) error {
+	if err := c.assignStretchMonsToZones(mons); err != nil {
+		return errors.Wrap(err, "failed to assign mons to zones")
+	}
+
+	// Enable the mon connectivity strategy
+	if err := client.EnableStretchElectionStrategy(c.context, c.ClusterInfo); err != nil {
+		return errors.Wrap(err, "failed to enable stretch cluster")
+	}
+
+	// Create the default crush rule for stretch clusters, that by default will also apply to all pools
+	if err := client.CreateDefaultStretchCrushRule(c.context, c.ClusterInfo, &c.spec, c.stretchFailureDomainName(), c.spec.Mon.StretchCluster.SubFailureDomain); err != nil {
```

Comment:
We pass `c.spec` so we don't need `c.spec.Mon.StretchCluster.SubFailureDomain`

Reply:
Funny how I ask everyone else to do it and then overlook it myself :)